python/tvm/contrib/hexagon/build.py

```diff
@@ -17,146 +17,320 @@
 
 """Defines top-level glue functions for building Hexagon."""
 
-import pathlib
+import abc
+import datetime
+import multiprocessing as mp
 import os
+import pathlib
+import signal
+import stat
 import subprocess
 from typing import Union
-import stat
-import datetime
 
 import tvm
 from ..._ffi import libinfo
 from .session import Session
 
 
-RPC_SERVER_FILES = ["tvm_rpc_android", "libtvm_runtime.so", "android_bash.sh"]
-
-HEXAGON_FILES = ["libhexagon_rpc_skel.so"]
-
-HEXAGON_RPC_DIR = None
+HEXAGON_RPC_LIB_DIR = os.environ.get("HEXAGON_RPC_LIB_DIR")
```

Comment:
maybe add default to be `None`?

Reply:
`None` is already the default "default value" in `get`:
```
>>> import os
>>> v = os.environ.get("NO_SUCH_VAR")
>>> print(v)
None
```